Vector.cpp

```diff
@@ -14,6 +14,9 @@ class Custom_Vector
     typedef int *iterator;
     typedef const int *const_iterator;
 
+    typedef int *riterator;
```

Comment:
Wrong implementation of reverse iterators. They can't be the same. The `operator++` implementation of the reverse iterator must be equal to `operator--` for the forward iterator.